packages/contentful-apps/redirects/src/locations/EntryEditor.tsx

```diff
@@ -0,0 +1,130 @@
+import { ModalLauncher, Stack } from '@contentful/f36-components'
+import { WorkbenchContent } from '@contentful/f36-workbench'
+import { useFieldValue } from '@contentful/react-apps-toolkit'
+import { useEffect, useRef, useState } from 'react'
+import { Form, ModalDelete, ModalEdit, Table } from '../components'
+import { Redirect } from '../types'
+import { arrayMove, countSlashes } from '../Utils'
+
+const Field = () => {
+  const [redirects = [], setRedirects] = useFieldValue<Redirect[]>('redirects')
+  const [editMode, setEditMode] = useState(false)
+  const formRef = useRef<any>()
+
+  useEffect(() => {
+    // In case redirects is anything other than array resets it
+    if (!Array.isArray(redirects)) setRedirects([])
+  }, [redirects])
+
+  const getRedirectIndex = (from: string): number => {
+    return redirects.findIndex((redirect: Redirect) => redirect.from === from)
+  }
+
+  const removeRedirect = (index: number): void => {
+    const tempArr = [...redirects]
+    tempArr.splice(index, 1)
+    setRedirects(tempArr)
+  }
+
+  // Find the position to add the new redirect using the '/' count
+  const findArrayPosition = (currentCount: number): number => {
+    for (let index = redirects.length - 1; index > 0; index--) {
+      const count = countSlashes(redirects[index].from)
+      if (currentCount <= count) {
+        return index + 1
+      }
+    }
+    return 0
+  }
+
+  const addRedirect = (redirect: Redirect): void => {
+    // In case it uses Netlify wildcard adds the redirect in order at bottom
+    if (redirect.from.endsWith('/*')) {
```

Comment:
We should discuss this logic, maybe together with Holger.

Netlify also supports placeholders which should come after the links without placeholders.
All other redirects are simply sorted at the start, which will be a mess for long lists.
I would prefer to have a list generally ordered by path or a possibility for the user to change the order (in that case the user would be responsible to place the wildcard matches after the static paths).
Letting the user do the order would also help with using this app for other deployment platforms, since the `*` only at the end is kind of netlify specific.
When we sort the rows, we need to ensure that the order makes sense for the target platform, which is problematic. But this kind of "random" sorting will be confusing I think.